EventOnFly/EventOnFly/DB/StoredProcedures/Service_Create.sql

```diff
@@ -0,0 +1,36 @@
+IF EXISTS (SELECT * FROM sys.objects WHERE type = 'P' AND name = 'Service_Create')
+DROP PROCEDURE [dbo].[Service_Create]
+GO
+
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+-- =============================================
+-- Author:      Sofia Huts
+-- Create Date: 04/21/2018
+-- Description: Create Service
+-- =============================================
+CREATE PROCEDURE [dbo].[Service_Create]
```

Comment:
rename procedure to uspCreateNewService (as in ProcedureName.cs) or rename corresponding value in ProcedureName enum